downloads/api/oai/shipcloud_v1_oai3.json

```diff
@@ -4120,6 +4120,35 @@
         },
         "description": "label specific definitions"
       },
+      "localized_attributes": {
+        "title": "localized_attributes",
+        "description": "Dynamic object to contain text in different languages.",
```

Comment:
This description isn't displayed by swagger-ui. Can we fix that somehow? Otherwise, I think we should remove it for now to avoid confusion in the future.

Reply:
Chances of fixing this are slim, better remove it